SU-SWS/stanford_basic
# D8CORE-1123: Enable dropdowns for desktop

# READY FOR REVIEW

# Summary
- This PR adds support for dropdown navigation menus at desktop resolutions for stanford_basic

# Review By (Date)
- end of sprint

# Urgency
- it's a feature for the 2.0 release.

# Steps to Test

1. Checkout this branch.
2. Clear your caches.
3. Add a page or two, and add menu links for them.
4. go to `/admin/structure/menu/manage/main` and set up some of your pages so they are sub-items of other menu itmes.
5. go to `/admin/appearance/settings/stanford_basic` and check the option for "Enable dropdowns for navigation menu"
6. Save your changes, clear your caches.  Visit any page and check to make sure the drop down menus work for you.

# Affected Projects or Products
- Does this PR impact any particular projects, products, or modules?

# Associated Issues and/or People
- Acceptance tests in the profile here: https://github.com/SU-SWS/stanford_profile/pull/349

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
